examples/grid_example/public/app.tsx

```diff
@@ -146,6 +168,34 @@ export const GridExample = ({ coreStart }: { coreStart: CoreStart }) => {
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
               <EuiFlexGroup gutterSize="xs" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButtonGroup
+                    legend={i18n.translate('examples.gridExample.layoutOptionsLegend', {
+                      defaultMessage: 'Layout options',
+                    })}
+                    options={[
+                      {
+                        id: 'responsive',
+                        label: i18n.translate('examples.gridExample.responsiveLayoutOption', {
+                          defaultMessage: 'Responsive layout',
+                        }),
+                        toolTipContent:
+                          'The layout adjusts when the window is resized. Panel interactivity, such as moving and resizing within the grid, is disabled.',
+                      },
+                      {
+                        id: 'fixed',
+                        label: i18n.translate('examples.gridExample.fixedLayoutOption', {
+                          defaultMessage: 'Fixed layout',
+                        }),
+                        toolTipContent: 'The layout does not adjust when the window is resized.',
+                      },
```

Comment:
d8a85fe8d5dd3240531776b45ce9ef0045817fae
I also changed the name of a passed prop to `accessMode: `VIEW` | `EDIT`. Let me know what you think 🙏🏼

Reply:
Can we make the default "Edit" mode? It's just more useful for the sake of the example - having to switch to edit mode every time I want to mess around with the actual layout engine functionality is annoying 😆 


> I also mentioned some of my doubts in Considerations section of the description of this PR. I would appreciate your opinion

I can address these in separate comments so that we don't blow up this thread :bow:

Reply:
Done here: 337c22f41f327eefdf6cc06ee0e47d77b1be1623